composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "liip/functional-test-bundle",
+    "name": "liip/test-fixtures-bundle",
     "description": "This bundles provides additional functional test-cases for Symfony applications",
```

Comment:
This description has to be changed.

Reply:
`This bundles enables efficient loading of Doctrine fixtures in functional test-cases for Symfony applications`